src/modules/task.mod.js

```diff
@@ -0,0 +1 @@
+export let Tasks = (localStorage.getItem('tasks') === null) ? [] : JSON.parse(localStorage.getItem('tasks'));
```

Comment:
- [ ]   Kindly remove this line of code and add it to the task-status module as it make it simple